2-mandatory/3-stocks.js

```diff
@@ -49,6 +57,17 @@ function getAveragePrices(closingPricesForAllStocks) {
 */
 function getPriceChanges(closingPricesForAllStocks) {
     // TODO
+    let closingPrices = [];
+
+    for (let i = 0; i < closingPricesForAllStocks.length; i++) {
+        const numberOfPrices = closingPricesForAllStocks[i].length - 1;
+        const firstDay = closingPricesForAllStocks[i].at(0);
```

Comment:
It's also possible here to use bracket notation twice. For example, `closingPricesForAllStocks[i][0]`.